components/browser/state/src/main/java/mozilla/components/browser/state/state/content/DownloadState.kt

```diff
@@ -42,7 +45,8 @@ data class DownloadState(
     val skipConfirmation: Boolean = false,
     val id: String = UUID.randomUUID().toString(),
     val sessionId: String? = null,
-    val createdTime: Long = System.currentTimeMillis()
+    val createdTime: Long = System.currentTimeMillis(),
+    val response: @RawValue Response? = null
```

Comment:
Ugh, that `Parcelize` is scaray here. Do we really need to pass this along as `Parcelable` anywhere?

Reply:
We were using it before, but now we don't required it. We could remove support for it.

Reply:
That would be great, I think. :)